rules/windows/privilege_escalation_tokenmanip_sedebugpriv_enabled.toml

```diff
@@ -0,0 +1,85 @@
+[metadata]
+creation_date = "2022/10/20"
+maturity = "production"
+min_stack_comments = "New fields added: required_fields, related_integrations, setup"
+min_stack_version = "8.3.0"
+updated_date = "2022/10/20"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies the creation of a process running as SYSTEM and impersonating a Windows core binary privileges. Adversaries
+may create a new process with a different token to escalate privileges and bypass access controls.
+"""
+from = "now-9m"
+index = ["winlogbeat-*", "logs-windows.*"]
+language = "eql"
+license = "Elastic License v2"
+name = "SeDebugPrivilege Enabled by a Suspicious Process"
+note = """## Setup
+
+Windows Event 4703 logs Token Privileges changes and need to be configured (Enable).
+
+Steps to implement the logging policy with with Advanced Audit Configuration:
+
+```
+Computer Configuration >
+Policies >
+Windows Settings >
+Security Settings >
+Advanced Audit Policies Configuration >
+Audit Policies >
+Detailed Tracking >
+Token Right Adjusted Events (Success)
+```
+"""
+references = [
+    "https://learn.microsoft.com/en-us/windows/security/threat-protection/auditing/event-4703",
+    "https://blog.palantir.com/windows-privilege-abuse-auditing-detection-and-defense-3078a403d74e",
+]
+risk_score = 47
+rule_id = "97020e61-e591-4191-8a3b-2861a2b887cd"
+severity = "medium"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Privilege Escalation"]
+timestamp_override = "event.ingested"
+type = "eql"
+
+query = '''
+any where event.provider: "Microsoft-Windows-Security-Auditing" and
+ event.action : "Token Right Adjusted Events" and
+
+ winlog.event_data.EnabledPrivilegeList : "SeDebugPrivilege" and
+
+ /* exclude processes with System Integrity  */
+ not winlog.event_data.SubjectUserSid : ("S-1-5-18", "S-1-5-19", "S-1-5-20") and
+
+ not winlog.event_data.ProcessName :
+         ("?:\\Windows\\System32\\msiexec.exe",
+          "?:\\Windows\\SysWOW64\\msiexec.exe",
+          "?:\\Windows\\System32\\lsass.exe",
+          "?:\\Windows\\WinSxS\\*",
+          "?:\\Program Files\\*",
+          "?:\\Program Files (x86)\\*",
+          "?:\\Windows\\System32\\MRT.exe",
+          "?:\\Windows\\System32\\cleanmgr.exe",
+          "?:\\Windows\\System32\\taskhostw.exe",
+          "?:\\Windows\\System32\\mmc.exe",
+          "?:\\Users\\*\\AppData\\Local\\Temp\\*-*\\DismHost.exe",
+          "?:\\Windows\\System32\\auditpol.exe",
+          "?:\\Windows\\System32\\wbem\\WmiPrvSe.exe",
+          "?:\\Windows\\SysWOW64\\wbem\\WmiPrvSe.exe")
+'''
+
+
+[[rule.threat]]
+framework = "MITRE ATT&CK"
+[[rule.threat.technique]]
+id = "T1134"
+name = "Access Token Manipulation"
+reference = "https://attack.mitre.org/techniques/T1134/"
+
```

Comment:
Could we add the subtechnique here? [Create Process with Token]( https://attack.mitre.org/techniques/T1134/002/)